style.css

```diff
@@ -508,6 +552,10 @@ ul.skills-list li:first-child {
   gap: 20px;
 }
 
+#contact form .desktop {
+  display: none;
+}
```

Comment:
- [ ] I would suggest you please use `color:#F55800;` for `Get in touch` buttons in contact form to match the design